app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="container w-50 mx-auto shadow p-3 mb-5 bg-white rounded">
+<h2 class="text-center">Sign up </h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+<div class="form-group">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+<div class="form-group">
+    <%= f.label :photo %><br />
+    <%= f.text_field :photo, autofocus: true, autocomplete: "photo" %>
+  </div>
+
+<div class="form-group">
+    <%= f.label :bio %><br />
+    <%= f.text_field :bio, autofocus: true, autocomplete: "bio" %>
+  </div>
+
+<div class="form-group">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+<div class="form-group">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+<div class="form-group">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up", class: 'btn-sm btn btn-success mt-2' %>
+  </div>
```

Comment:
- [ ] Just like the previous reviewer stated, in keeping with the wireframe layout, kindly ensure that the input boxes take up width of their container.
- [ ] Please also make the signup button have the same width as the input boxes just like in the wireframe template attached below 👇 
## Login Wireframe
![image](https://github.com/techEmprez/Blog_app/assets/46839548/582c39c6-e4a5-43f5-ab16-ea751f4c504b)
## Your Login Page
![image](https://github.com/techEmprez/Blog_app/assets/46839548/2f9189e5-0ca9-4205-ab7a-80841e90a585)

Reply:
I have removed and implemented all necessary changes.
Please reviewer, kindly reach out via slack or preferrable a direct zoom call if response is delayed for additional changes.
Implemented all styling too
Please.... Please. Thank you.